lib/src/quality/apps/text/links.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import requests
+from malevich.square import DF, Context, processor, scheme
+from pydantic import BaseModel
+
+
+@scheme()
+class Links(BaseModel):
+    link: str
+
+@processor()
+def asser_links(df: DF[Links], context: Context):
+    """
+        Check if the links are valid (do not return error status codes)
+
+        Input:
+          DF[Links] - DataFrame with one column named 'link'.
+
+        Output:
+          DF[Links] - DataFrame with one column named 'link' containing
+          valid or invalid links depending on 'filter_links' option.
+
+        Options:
```

Comment:
Should be "Configuration" and also level-two header. Make lists

```
## Configuration
     - field(type, optional|required): description
     - ...
     
```

Do not use 'returns'. Imagine you are reading it from UI. Better to write "If set to True, the result will be filtered links, otherwise a collection of errors"

Reply:
Done